content/docs/command-reference/remote/modify.md

```diff
@@ -589,6 +589,9 @@ more information.
     to the remote in the form: `custom_auth_header: password`.
```

Comment:
```suggestion
    to the remote in the form: `custom_auth_header: <value>`.
```

Reply:
UNRELATED: @efiop what do you think about changing this BTW ^, using `password` as value here seems confusing, since `password` is also a field you can set separately of `auth`. Unless I'm confused and that's the point...

Reply:
That's precisely what you should do: set password in the config. There is a sentence clarifying that right after this one. Using `<value>` would be confusing here, though.

Reply:
OK thanks. Leaving this as is then.